### ISSUE TITLE
May I ask if it is possible to deploy to the server?

### ISSUE THREAD

**Issue by DreamerYinYu** [X-niper/UniTalker#33]
Hello, may I encapsulate UniTalker into a Python project, deploy it to a server, and provide services for the Unreal Engine project to drive digital human mouth shapes? Sorry, my question may be more basic as I am a beginner. Thank you

**Comment by X-niper** [comment]
> Hello, may I encapsulate UniTalker into a Python project, deploy it to a server, and provide services for the Unreal Engine project to drive digital human mouth shapes? Sorry, my question may be more basic as I am a beginner. Thank you

Yes, our team have tried this and it needs some engineering knowledeges.
